src/pages/CreatePollPage.js

```diff
@@ -75,6 +74,19 @@ class CreatePollPage extends Component {
             isValid = false
             errors["question"] = "Question cannot be empty"
         }
+        if(!fields["votingStart"]){
+            isValid = false
+            errors["votingStart"] = "Voting start must be specified"
+        }
+        if(fields["votingEnd"] <= fields["votingStart"]){
```

Comment:
This is not necessary as the datetimepicker has its own error- checks and messages. You can set this in "minDate". The other checks can be done by setting "required" to true.